aloha-core/src/main/scala/com/eharmony/aloha/models/exploration/BootstrapModel.scala

```diff
@@ -0,0 +1,108 @@
+package com.eharmony.aloha.models.exploration
+
+import com.eharmony.aloha.factory.{ModelFactory, ModelParser, ParserProviderCompanion}
+import com.eharmony.aloha.id.ModelIdentity
+import com.eharmony.aloha.models.{Model, BaseModel}
+import com.eharmony.aloha.score.Scores.Score
+import com.eharmony.aloha.score.basic.ModelOutput
+import com.eharmony.aloha.score.conversions.ScoreConverter
+import com.eharmony.aloha.score.conversions.ScoreConverter.Implicits.IntScoreConverter
+import com.eharmony.aloha.semantics.Semantics
+import com.eharmony.aloha.semantics.func.GenAggFunc
+import com.mwt.explorers.BootstrapExplorer
+import com.mwt.policies.Policy
+
+import scala.collection.JavaConversions.seqAsJavaList
+import scala.collection.{immutable => sci}
+
+case class NumberedPolicy(index: Int) extends Policy[sci.IndexedSeq[Int]] {
+  override def chooseAction(actions: sci.IndexedSeq[Int]): Int = actions(index)
+}
+
+/**
+  * Created by jmorra on 2/26/16.
+  */
+case class BootstrapModel[A, B](
+  modelId: ModelIdentity,
+  models: sci.IndexedSeq[Model[A, Int]],
+  salt: GenAggFunc[A, Long],
```

Comment:
Yea, I totally said I'd comment this very well and then didn't.  I'll add a bunch of comments.